data-raw/simu_test_gs_design_combo.R

```diff
@@ -41,31 +41,31 @@ sim_gsd_pMaxCombo_exp1_H0 <- function(N = ceiling(454.60),
   failRates$failRate <- failRates$failRate * (failRates$hr + 1) / 2
   failRates$hr <- 1
 
-  failRates0 <- tibble::tibble(
+  failRates0 <- tibble(
```

Comment:
This script is a bit of a mess. As it is I don't think it can even run. It never loads {dplyr} or {tibble} despite calling functions without the qualifier. It also calls a function `tensurv()`, which I couldn't find. This is a standalone script, not part of the R package, so we can't assume that the functions in `NAMESPACE` are available to it. If that is the behavior we want, we should add `devtools::load_all()` to the top of it.

Reply:
The change in this script was accidental. I'll revert.